docs/src/modules/branding/BrandingSupportTheProject.tsx

```diff
@@ -0,0 +1,151 @@
+import FinanceIcon from 'docs/src/modules/branding/icons/Finance';
+import HelpIcon from 'docs/src/modules/branding/icons/Help';
+import OpenCollectiveIcon from 'docs/src/modules/branding/icons/OpenCollective';
+
+interface SupportCardProps {
+  color?: 'primary' | 'info';
+  icon?: React.ReactNode;
+  children?: React.ReactNode;
+  title?: string;
+}
+
+const SupportCard = styled((props: SupportCardProps) => {
+  const { color, icon, title, children, ...other } = props;
+  return (
+    <Card elevation={0} {...other}>
+      <CardHeader avatar={<Avatar aria-label={title}>{icon}</Avatar>} />
+      <CardContent>
+        <Typography variant="h3">{title}</Typography>
+        <Typography component="div">{children}</Typography>
+      </CardContent>
+    </Card>
+  );
+})<SupportCardProps>(({ color = 'primary', theme }) => ({
+  '& [class*="MuiAvatar-root"]': {
+    background: color === 'info' ? theme.palette.vividBlue : theme.palette.primary.main,
+    width: 80,
+    height: 80,
+  },
+}));
+
+export default function BrandingSupportTheProject() {
+  return (
+    <Container sx={{ mt: [10, 18], mb: [12, 20] }}>
+      <Typography variant="h2">Material-UI is awesome.</Typography>
+      <Typography variant="h2">
+        How can <UnderlinedText>I support the project?</UnderlinedText>
+      </Typography>
+      <Typography sx={{ mb: 4 }}>There are many ways to support Material-UI:</Typography>
+      <Grid container spacing={2}>
+        <Grid item xs={12} md={6}>
+          <SupportCard icon={<QuestionAnswerIcon fontSize="large" />} title="Spread the word">
+            <Typography sx={{ mt: 2 }}>
+              Evangelize Material-UI by linking to{' '}
+              <Link href="https://material-ui.com">material-ui.com</Link> on your website, every
+              backlink matters! Follow us on{' '}
+              <Link href="https://twitter.com/MaterialUI">Twitter</Link>, like and retweet the
+              important news. Or just talk about us with your friends.
+            </Typography>
+          </SupportCard>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <SupportCard
+            color="info"
+            icon={<FeedbackIcon fontSize="large" />}
+            title="Give us feedback"
+          >
+            <Typography sx={{ mt: 2 }}>
+              Tell us what we&apos;re doing well or where we can improve. Please upvote (
+              <span role="img" aria-label="Thumbs up emoji">
+                👍
+              </span>
+              ) the issues that you are the most interested in seeing solved. {/* TODO: add link */}
+              <Link href={'/'}>Give us feedback</Link>
+            </Typography>
+          </SupportCard>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <SupportCard
+            color="info"
+            icon={<ChangesIcon fontSize="large" />}
+            title="Make changes happen"
+          >
+            <ul>
+              <li>
```

Comment:
It looks like we can update the `theme.components.MuiCssBaseline` of `BrandingRoot.tsx` to handle the `<ul>` and `<li>` node style globally. For instance, `ul { padding-left: 32px; }` could be enough.

Reply:
We could add `list-style: none`, anything apart from that may not be the best default, as we can different use-cases

![image](https://user-images.githubusercontent.com/4512430/104427638-c76fc600-5583-11eb-90db-e6e688b2003a.png)

![image](https://user-images.githubusercontent.com/4512430/104427663-d0609780-5583-11eb-9320-6786e62ce0a9.png)

![image](https://user-images.githubusercontent.com/4512430/104427677-d6ef0f00-5583-11eb-8a7f-2652c9679523.png)

Reply:
I've added only the `list-style: none` on the CssBaseline, but added this override on the `BrandingCard` used for this particular list:

```
sx={{
  ul: {
    position: 'relative',
    ml: 0,
    pl: '1.2em',
  },
  'ul li:before': {
    content: '"-"',
    position: 'absolute',
    left: 0,
  },
}}
```